libtenzir/src/tql2/parser.cpp

```diff
@@ -641,8 +640,109 @@ class parser {
           result.push_back('\t');
         } else if (x == 'n') {
           result.push_back('\n');
+        } else if (x == 'r') {
+          result.push_back('\r');
+        } else if (x == 'b') {
+          result.push_back('\b');
+        } else if (x == 'f') {
+          result.push_back('\f');
+        } else if (x == 'v') {
+          result.push_back('\v');
+        } else if (x == 'a') {
+          result.push_back('\a');
         } else if (x == '0') {
           result.push_back('\0');
+        } else if (x == 'u' || x == 'U') {
+          // Unicode escape sequence: \uXXXX or \UXXXXXXXX
```

Comment:
It's also possible to just have `\u{…}` instead with 1-6 hex digits. Would that be better?

Reply:
That's easily added, will do.